cluster/gce/util.sh

```diff
@@ -1357,6 +1361,9 @@ KUBE_POD_LOG_READERS_GROUP: 2007
 KONNECTIVITY_SERVER_RUNASUSER: 2008
 KONNECTIVITY_SERVER_RUNASGROUP: 2008
 KONNECTIVITY_SERVER_SOCKET_WRITER_GROUP: 2008
+CLOUD_CONTROLLER_MANAGER_RUNASUSER: 2009
```

Comment:
Wondering why this is missing from https://github.com/kubernetes/cloud-provider-gcp/blame/992c1c33f1be3f5204934e02b8488d117b50dd1c/cluster/gce/util.sh#L1337 ?